microservices/download/utils.js

```diff
@@ -110,6 +114,17 @@ async function findLink(os, type) {
 
     let releases = await doRequest();
 
+    if (!Array.isArray(releases)) {
+      console.error("GitHub Returned invalid data on release request!");
+      console.error(releases);
+
+      return {
+        ok: false,
+        code: 500,
+        msg: "Request to GitHub for releases failed."
+      };
+    }
+
```

Comment:
Nice. 👍 

This particular situation wasn't our fault it seems, per your diagnosis/investigation/recap on Discord, however as shown in this diff, I can see that we can log the situation clearly and even handle it more gracefully on our end, by anticipating this data may be coming in broken or missing. Seems like a good move.